recipes/bitserializer/all/conanfile.py

```diff
@@ -94,14 +80,7 @@ def package_id(self):
             self.info.clear()
 
     def validate(self):
-        if self.settings.compiler.get_safe("cppstd"):
-            check_min_cppstd(self, self._min_cppstd)
-
-        minimum_version = self._compilers_minimum_version.get(str(self.settings.compiler), False)
-        if minimum_version and Version(self.settings.compiler.version) < minimum_version:
-            raise ConanInvalidConfiguration(
-                f"{self.ref} requires C++{self._min_cppstd}, which your compiler does not support.",
-            )
+        check_min_cppstd(self, 17)
```

Comment:
This is definitely simple, but compilers which claim to support C++ 17 may not have all features which requires the library. I would prefer to have more granular check.

Reply:
Thanks for the feedback - while it's true that some compilers might not actually have the necessary C++17 support, we're trying to move away from blanket checks for all compiler versions like this.

Of course, we will accept specific compiler version checks for those that -std=c++17 does not mean it would compile on those, but in most cases we've found that unless a proper message is given, the compiler version checks end up being a bit too confusing

So the question here would be if the old checks for those compilers accepted C++ 17 but failed otherwise, or it any of them were a "C++17" was introduced here?

Thanks!

Reply:
The old checks were required for check versions of compilers which supports the library (requirements from the official [README.md](https://github.com/PavelKisliak/BitSerializer/blob/master/README.md#requirements)). The TestPackage from recipe may compile with older versions, but the library will not work on the end user side.

The library's CMake script has all necessary checks, but it's not used when enabled only header-only components.
So, I think that these checks of compiler version are still necessary for early inform user that he has wrong setup.

Reply:
Hello! Can we move forward with this PR?
Please let me know if there is anything else I can do.
Are there any objections if I bring back compilers checking?